src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php

```diff
@@ -174,8 +174,7 @@ public function getResources(): array
     private function resetPagination()
     {
         $con = $this->connection->getResource();
-        $this->controlPagedResultResponse($con, 0, '');
-        $this->serverctrls = [];
+        $this->controlPagedResultResponse($con, null, '');
```

Comment:
shouldn't we skip calling this method instead here (and keep `$this->serverctrls = [];`?)
see my other comment

Reply:
I think so. I just wanted to not break anything since I can't test it.